runtime/lua/vim/lsp/util.lua

```diff
@@ -263,10 +263,9 @@ end
 -- Sort by CompletionItem.sortText
 -- https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_completion
 local function sort_completion_items(items)
-  if items[1] and items[1].sortText then
-    table.sort(items, function(a, b) return a.sortText < b.sortText
-    end)
-  end
+  table.sort(items, function(a, b)
+    return (a.sortText or a.label) < (b.sortText or b.label)
```

Comment:
While we're here, perhaps we should also add `or ''` to both of these, just so that we don't completely die on the comparison if we do for some reason have nil for both. What do you think about that?

Reply:
It should always be there so if it isn't I think it might be a good idea to error out so that the bug causing it to be nil can be fixed.

Reply:
I don't feel super strongly about it. We can leave it the way it is for now.

Reply:
According to the specifications of lsp, this should not cause an error.  When an error occurs, I think that the specification has been changed or the server is a bug, so I can leave it as it is.